selftests/checkall

```diff
@@ -7,11 +7,11 @@ run_rc() {
         GR=1
     fi
 }
-run_rc 'inspekt lint'
+run_rc 'inspekt --exclude=.git lint'
```

Comment:
@Andrei-Stepanov inspektor `--exclude` is used explicitly here because it would feel akward to list `.git/*` in `.gitignore` (or similar files).  When you use GIT's `rerere`, it creates some files that would be picked up by inspektor.